scene/gui/rich_text_label.cpp

```diff
@@ -5650,10 +5650,15 @@ void RichTextLabel::set_text(const String &p_bbcode) {
 }
 
 void RichTextLabel::_apply_translation() {
+	// If `text` is empty, it could mean that the tag stack is being used instead. Leave it be.
```

Comment:
How do we handle updated translations when using the tag stack then?

Reply:
I checked and the tooltips are still being translated, so it's done at the time of pushing the text.